sandbox/concurrent_bert_etl/app_all_lambdas/jobs.py

```diff
@@ -0,0 +1,385 @@
+"""This module defines all the Lambda functions to be deployed to AWS.
+The function name would be AWS Lambda function name as well. Everything to be
+packaged to Lambda must be contained *inside* the function. Module level
+imports are for local testing or packaging use only.
+
+Use ``bert.constants.AWS_LAMBDA_FUNCTION`` to check if function is running
+locally or in AWS.
+
+When the ``DEBUG`` environment variable is set to ``true``, test event can
+be run from Lambda dashboard, but it will not call the next Lambda because
+it will not write to DynamoDB. When ``DEBUG`` is ``false``, the whole thing
+will run but hardcoding the test case in the first function is necessary.
+
+"""
+from bert import binding, constants, utils
+
+
+@binding.follow('noop')
+def bert_tess_fullframe_main_0():
+    """Farms out tasks for a list of TIC IDs."""
+
+    import os
+
+    import boto3
+
+    work_queue, done_queue, ologger = utils.comm_binders(
+        bert_tess_fullframe_main_0)
+
+    s3 = boto3.resource('s3')
+    homedir = os.environ.get('HOME')
+
+    # NOTE: See the note about DEBUG in module docstring.
+    # TODO: Hardcode test case here.
+
+    # Example event:
+    # {
+    #   "bucket": "my_bucket",
+    #   "key": "my_tic_id_list.csv"
+    # }
+    for event in work_queue:
+        bucketname = event['bucket']
+        bucket = s3.Bucket(name=bucketname)
+        key = event['key']
+        basename = os.path.basename(key)
+        filename = os.path.join(homedir, basename)
+
+        try:
+            ologger.info(f'Attempting to download {key} from {bucketname}')
+            bucket.download_file(
+                key, filename, ExtraArgs={"RequestPayer": "requester"})
+        except Exception as exc:
+            ologger.error(f'{basename}: {str(exc)}')
+            continue
+        else:
+            ologger.info(f'Parsing {filename}')
+            with open(filename) as fin:
+                for line in fin:
+                    row = line.split()[0].split(',')
+                    tic_id = row[0]
+                    ologger.info(f'Processing {tic_id}')
+                    d = {'tic_id': tic_id,
```

Comment:
TODO: Might have to skip header. How about skip all the lines that start with `#`?